Bottom: 8f7fc643fedd2f9c0e8b86abba1aa1f459dadc6a
Top:    61d67da51d9155849464fd52ecce54d5d255a81c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-28 03:58:13 +0000

Disable debug hack


---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 0a325f8..52182c0 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -152,7 +152,7 @@ step_do_export (HID_Attr_Val * options)
   if (filename == NULL)
     filename = "pcb-out.step";
 
-#if 0
+#if 1
   board_outline_list = object3d_from_board_outline ();
 #else
   board_outline_list = NULL;
